modules/onscreengui/src/renderproperties.cpp

```diff
@@ -46,13 +46,22 @@ void renderTooltip(Property* prop) {
     }
 }
 
-void executeScript(const std::string& id, const std::string& value) {
+void executeScriptSingle(const std::string& id, const std::string& value) {
```

Comment:
Yes, if I write keybindings I'll do it with the `setPropertyValueSingle` function. I think it is good to have the option in case we don't want to pay the cost of the regex